docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "0.1"
```

Comment:
3.8 is the latest https://docs.docker.com/compose/compose-file/

Reply:
thanks! this is still work in progress though.